src/theme/styles/colours.ts

```diff
@@ -100,11 +98,17 @@ export const LIGHT_COLOURS = {
   labelBgOpen: '#77838F1A',
```

Comment:
Plan to address these in a future PR:

`labelTextopen`
`labelBgOpen`

Ideally by again falling back to a shared color (e.g. `grey` and `greyOpacity`).